src/controllers/comment-controller.js

```diff
@@ -0,0 +1,60 @@
+const Comments = require('../models/comments');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const Comments = require("../models/comments");
```